fe/fe-core/src/main/java/com/starrocks/sql/ast/DistributionDesc.java

```diff
@@ -61,6 +61,10 @@ public static DistributionDesc read(DataInput in) throws IOException {
             DistributionDesc desc = new HashDistributionDesc();
             desc.readFields(in);
             return desc;
+        } else if (type == DistributionInfoType.RANDOM) {
```

Comment:
Do we need support to convert the hash distribution to random distribution?

Reply:
using CTAS now